charts/gardener/operator/values.yaml

```diff
@@ -56,6 +56,7 @@ config:
     enableContentionProfiling: false
   featureGates:
     DefaultSeccompProfile: true
+    VPAInPlaceUpdates: true
```

Comment:
This will also enable the feature gate for anybody deploying the operator using these charts, if you want to enable it only for the local setup, you can use something like - https://github.com/gardener/gardener/blob/e07aedcd611924f8a60af58ee9284883141497ea/skaffold-operator.yaml#L1019